### ISSUE TITLE
had a hard time to understand how negative log likelihood is calculated in the regression example

### ISSUE THREAD

**Issue by wmmxk** [Kyushik/Predictive-Uncertainty-Estimation-using-Deep-Ensemble#1]
I was trying to understand "loss = tf.reduce_mean(0.5*tf.log(output_sig_pos) + 0.5*tf.div(tf.square(y - output_mu),output_sig_pos))". Could you elaborate more on this?

**Comment by wmmxk** [comment]
[Here is the link for the Jupyter notebook](https://github.com/Kyushik/Predictive-Uncertainty-Estimation-using-Deep-Ensemble/blob/master/Ensemble_Regression_RealData.ipynb)

**Comment by Kyushik** [comment]
Hello. I used the loss function, which is equation (1) of the following paper. 
https://papers.nips.cc/paper/7219-simple-and-scalable-predictive-uncertainty-estimation-using-deep-ensembles.pdf

**Comment by Leezzzz** [comment]
Hello, I'm having some problems.There's always a warning "raise ValueError('There is Nan in loss')",but i don't kown how to solved it. Could you help me?

**Comment by yang-ni-yn** [comment]
Hi, I am curious about how you decide the constant value at the end of that loss function? Thanks.
